### DISCUSSION TITLE
missing ootb csv import for data objects

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#16751] [Q&A / Support] [answered]
hi, can someone tell me why I don't have a CSV import option as explained here: https://medium.com/pimcore-development-hub/how-to-export-and-import-data-objects-in-pimcore-8912efdd47d6 
![grafik](https://github.com/pimcore/pimcore/assets/111425602/eea63356-0c07-4b63-9fe3-b3e9042d0bec)

**Comment by fashxp** [comment, marked as answer]
feature got replaces by data importer: https://pimcore.com/docs/platform/Data_Importer/
